examples/images/cifar_random_patch.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+#Set environment variables
+: ${KEYSTONE_MEM:=4g}
+export KEYSTONE_MEM
+
+KEYSTONE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/../..
```

Comment:
I dunno, @shivaram is the bash ninja. This is the top answer on stack overflow for "current bash script directory." :)